src/teacup.coffee

```diff
@@ -174,6 +176,13 @@ class Teacup
     @raw contents
     @raw "</#{tagName}>"
 
+  scriptTag: (tagName, args...) ->
+    {attrs, contents} = @normalizeArgs args
+    @raw "<#{tagName}#{@renderAttrs attrs}>"
+    @raw @scriptescape contents
```

Comment:
Oh found it: https://www.owasp.org/index.php/XSS_(Cross_Site_Scripting)_Prevention_Cheat_Sheet#RULE_.233.1_-_HTML_escape_JSON_values_in_an_HTML_context_and_read_the_data_with_JSON.parse